.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,58 @@
+name: Feature Request
+description: Suggest an idea
+labels:
+  - enhancement
+  - to-triage
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please describe the use-case you have. This will better help us understand the context in which you're looking for a new feature.
+  - type: textarea
+    id: use-case
+    attributes:
+      label: Use case
+      description: |
+        Please describe the use-case you have. This will better help us understand the context in which you're looking for a new feature.
+      placeholder: Describe the use-case here
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Use case
```

Comment:
It looks like a duplicate 
Should it be "Expected solution" ?

Reply:
Good catch. I copy-pasted the forms from https://github.com/jOOQ/jOOQ/blob/main/.github/ISSUE_TEMPLATE/feature_request.yaml :)